server/command/command.go

```diff
@@ -0,0 +1,63 @@
+package command
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/mattermost/mattermost/server/public/pluginapi"
+)
+
+type Handler struct {
+	client *pluginapi.Client
+}
+
+type Command interface {
+	Handle(args *model.CommandArgs) (*model.CommandResponse, *model.AppError)
```

Comment:
Can we expunge `model.AppError` in favour of `error`? I realize that requires a shim in the actual hook, but it would be ideal not to propagate this anywhere we don't strictly need it.

Reply:
Yeah good catch